desktop_styles.css

```diff
@@ -0,0 +1,309 @@
+@font-face {
+  font-family: Inter;
+  src: url(fonts\Inter-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts\Roboto-Regular.ttf);
+}
+
+#home-bg-img {
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 0;
+  width: 100vw;
+  height: 97vw;
+  background-image: url("images/Header-llustration-desktop.png");
+  z-index: -1;
+}
+
+#appbar,
+#hamburger-icon,
+#logo-placeholder,
+#name,
+#form-bg1 {
+  display: none;
+}
+
+#nav-list {
+  display: flex;
+  gap: 8px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+  list-style-type: none;
+}
+
+#nav-list li {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  color: #344563;
+  padding: 10px 12px;
+}
+
+#toolbar {
+  width: 100vw;
+  height: 72px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#hamburger-cont {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 72px;
+  margin-right: 0;
+}
+
+#name-txt {
+  overflow-wrap: break-word;
+  width: 700px;
+  display: block;
+  text-align: center;
+}
+
+#job-txt {
+  overflow-wrap: break-word;
+  width: 700px;
+  display: block;
+  text-align: center;
+}
+
+#home {
+  width: 66vw;
+}
+
+#message-cont {
+  width: 62vw;
+}
+
+#works {
+  height: auto;
+  top: 73.5vw;
+  width: 100vw;
+}
```

Comment:
there is only design for 1440px width property. What should I do ? especially for the works section. I really don't knoıw how can we do it.  I guess you need to compare my design and figma for 1440px screen width.

Reply:
Hi @s2jack, Figma design is just for reference and I already have attached the project requirement according to which you must have to implement the *Figma* design no matter *1440px* on >= 768px screen size. Because I am pretty confident that you used all the CSS styles for desktop design from Figma which is why your design is fixed for 1440px screen size but you shouldn't follow the Figma styles except the colour, font family, hover effect and a few others according to what makes sense. Now, the answer to your problem. Well, if you don't know then it is the right time to take a challenge for yourself and try to learn and implement the changes as I requested because in the upcoming projects especially during **Final Capstone Project** you will even not get the `Figma` design you will only get a design which nothing but a photo on behance website, which you need to follow in order to complete project. So if it is difficult for you now then it will be difficult during Final Capstone project too, but if you learn and implement the things now then *Final Capstone Project* will be like a cat walk. Any other question please feel free to mention @skyv26 and ask the question without hesitation, I am open to answer all your doubts with best possible suggestions, advices and with 100% satisfactory solution.